youtube_dl/YoutubeDL.py

```diff
@@ -161,6 +162,7 @@ class YoutubeDL(object):
     playlist_items:    Specific indices of playlist to download.
     playlistreverse:   Download playlist items in reverse order.
     playlistrandom:    Download playlist items in random order.
+    playlistprompt:    Ask for each playlist file to download
```

Comment:
This is the description of programmatic interface. How this option is supposed to be used via it?